.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
```

Comment:
I'm seeing E203 and W503 suggested [here](https://black.readthedocs.io/en/stable/compatible_configs.html#flake8). Can you post a reference for the additional errors/warnings that would be ignored? I'm tempted to include this link as a comment in the flake8 config.

Reply:
I've posted a link to the reference I used in the comment above.

I realized that the "strictness" that is recommended in the reference is pretty severe. I've removed the `select` key which selects the `pystyleguide`, `PyFlakes`, etc. [error codes](https://flake8.pycqa.org/en/2.6.0/warnings.html) and ignores a select subset of them as above. I wasn't sure how the [McCabe complexity threshold](https://flake8.pycqa.org/en/2.5.5/config.html#settings) (`max-complexity`) is important, so I've removed that requirement as well.